udata_front/theme/gouvfr/datagouv-components/src/components/ResourceAccordion/EditButton.vue

```diff
@@ -9,20 +9,36 @@
 </template>
 
 <script setup lang="ts">
+import { computed } from "vue";
 import { config } from "../../config";
 import { useI18n } from "vue-i18n";
+import { CommunityResource, Resource } from "../../types/resources";
 
 type Props = {
   datasetId: string,
   isCommunityResource?: boolean,
-  resourceId: string,
+  resource: Resource | CommunityResource,
 };
 
 const props = withDefaults(defineProps<Props>(), {
   isCommunityResource: false,
 });
 
 const { t } = useI18n();
-const resourceType = props.isCommunityResource ? 'community-resource' : 'resource';
-const adminUrl = `${config.admin_root}dataset/${props.datasetId}/${resourceType}/${props.resourceId}`;
+
+const adminUrl = computed(() => {
+  if (config.new_admin) {
+    if ('owner' in props.resource && props.resource.owner) {
+      return `${config.admin_root}users/${props.resource.owner.id}/community-resources/#${props.resource.id}`
+    } else if ('organization' in props.resource && props.resource.organization) {
+      return `${config.admin_root}organization/${props.resource.organization.id}/community-resources/#${props.resource.id}`
```

Comment:
Ideally, having a community resource link without the user or organization in path (as in the legacy admin) would make it easier (and it would be aligned with other objects), right?

Reply:
It's a little bit harder because it's a modal and not a full page… But I can make it work…

Reply:
It's not necessarily worth the trouble now, it may be done when we migrate the datasets page?

Reply:
Ok